cfgov/jinja2/v1/_includes/organisms/bureau-structure.html

```diff
@@ -0,0 +1,334 @@
+            <span class="o-bureau-structure_legend_definition">
+                Position is not part of the CFPB director’s office
+            </span>
+        </li>
+</section>
+{%- endmacro %}
+
+
+{# ==========================================================================
+
+   _footer()
+
+   ==========================================================================
+
+   Description:
+
+   Renders an org chart footer:
+
+   ========================================================================== #}
+{% macro _footer() -%}
+<aside class="block
+              block__padded-top
+              block__border-top
+              o-bureau-structure_footer">
+    <div class="content-l content-l__main">
+        <div class="content-l_col content-l_col-1-2"
+             data-qa-hook="org-chart-download-info">
+            <h2 class="h3">Download a copy</h2>
+            <p>
+                Download a printable, PDF copy of this organizational chart. This version
+                is expanded to show all offices.
+            </p>
+            {# TODO Change to PDF download URL when it's available. #}
+            <a class="btn"
+               href="{{ image(value.download_image, 'original').url }}">
+                <span class="btn_icon__left cf-icon cf-icon-save"></span>
+                Download PNG
+            </a>
+        </div>
+        <div class="content-l_col content-l_col-1-2">
+            <h2 class="h3">Request speaking info</h2>
+            <p>
+                Ask a CFPB employee to be involved in a forum, publication, discussion, or
+                other event; or to inquire about any CFPB events.
+            </p>
+            <span class="cf-icon cf-icon-email list_icon"></span>
+            <span class="list_text">
+                <a class="list_link"
+                   href="mailto:invitations2cfpb@consumerfinance.gov">
+                    invitations2cfpb@consumerfinance.gov
+                </a>
+            </span>
+        </div>
+    </div>
+</aside>
+{%- endmacro %}
+
+
+{# ==========================================================================
+
+    bureau_structure.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a Bureau Structure organism when given:
+
+   value:         An object used to customize the markup.
+
+   value.director: A string with the title for the header slug.
+
+   value.text:     The text used within the description markup.
+
+   value.gd_code:  A GovDelivery code for a specified mailing list.
+
+   ========================================================================== #}
+<div class="o-bureau-structure">
+    <link rel="stylesheet" href="{{ static('css/slick.css')}}">
+    <h1>Bureau Structure</h1>
+    {% if value.last_updated_date %}
+    {% import 'macros/time.html' as time %}
+    <p>
+        Org chart last updated:
+        {{ time.render( value.last_updated_date, {'date': true} ) }}
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2532/commits/861674d42470d1ebe15c3ac204e66b4fa58f741c.